packages/game-bridge/README.md

```diff
@@ -22,6 +22,20 @@ Note: The above build will bundle the index files in the following locations (re
 - Unity: `./dist/unity/index.html`
 - Unreal: `./dist/unreal/index.js`, `./dist/unreal/index.js.map`
 
+#### Building for Production / Release
+
+During the build step for game-gridge, placeholder SDK version strings are replaced with the actual SDK version strings. To ensure the correct version strings are packaged, as opposed to the local commit hash, which could be different than the latest release tag, the following process should be followed:
+
+1. Make/push changes to game bridge
+2. Merge to main
+3. Release new version of the SDK (using the `Publish to NPM` workflow)
+   - This creates a new GitHub tag / release
+4. Pull `main` locally after release workflow completes
```

Comment:
Even after pulling `main, ' should we call out that they should ensure that the commit is tagged with a version?

Because I could release v1.60.0, pull `main`, and because other PRs got merged in, I could be X commits ahead of v1.60.0.

Reply:
Also, when running game bridge `yarn build`, could we check that the commit it's on has a tag?

Reply:
Good call out. Should check out the tag and build instead.